teams/developer-relations/guide-to-being-a-manager/README.md

```diff
@@ -41,12 +92,45 @@ This checklist will be posted within the issue and should all be completed withi
   * [ ] Ensure General Review Checklist is being updated based on the developers' progress
 * [ ] Adjust timeline in Zenhub Calendar
+* [ ] Weekly Team Sync
+* [ ] 1:1s with devs
+
+## _Weekly Manager Checklist Documentation \[Owen\]_
+
+### Deadlines:
+
+Friday: Friday deadlines are the fist draft deadlines for your team. This means that your team members should have finished their issues and submitted a pull request to your branch for review. You should take the next few days to work on any problems that come up after review. 
+
+Sunday: Sunday deadlines are the final draft deadlines for you team. This means that you should submit a pull request to master with your team's completed work for the week. There will also be a form that documents your team's work for the week that is due. 
+
+Tuesday: Tuesday deadlines are for the issues that need to assigned to your team. 
+
+### Issues:
+
+Issues are unresolved problems that developers need to address. As a manager, it is your job to raise issues for your developers and assign them each week. You should make sure that there are enough issues each week to be worked on, raising issues as you review your developer's work as well as in line for your long-term goals.
+
+There are also other issues that should be raised and given as first-timer issues. These issues are generally less complicated resource wise compared to other issues. This does not mean that they are less difficult content wise.
+
+All issues addressed should fully address the long-term plans set up!
+
+### Milestones:
+
+Milestones are used to organize your weekly team's weekly tasks. Issues are placed into your milestones to keep track of your team's work for the week. To create a milestone go to the issues tab in GitHub and there should be a milestones button to the left of the new issue button. Here you can create new milestones as assign tasks to your milestones. In addition your milestones should follow the naming convention of your team-week of- Tuesday of week. 
+
+### Adjustments:
+
+After all your weekly tasks are completed, assess your long-term plan and make any changes necessary. If a part of your activity is completed, assign the appropriate epic points. If there was not as much progress as originally thought, modify the long-term deadlines. Adjust overall long-term plan according to the week's progress and any feedback from developers. 
+
+### Github Projects:
+
+Github Projects are another source of organization for your long-term plans. You can use them to group issues of your long-term plan for a module and track the progress of completion. 
 
 ## Pull Request Checklist
 
 Every week, there are a couple essential things that every manager should take care in pull requests. This checklist applies to both pull requests from developers' branches to managers' branches and from managers' branches to `master`.
 
 * [ ] **Branch updated from `master`**
+  * [ ] Resolving all Merge Conflicts
```

Comment:
I think you should talk about ways to avoid merge conflicts with your team in the first place; making sure people update to master/your branch every week, etc, ensuring that your team members are always using GitHub correctly